src/mmrelay/meshtastic/plugins.py

```
@@ -0,0 +1,153 @@
+                    plugin_timeout,
+                    loop,
+                )
+
+                if found_matching_plugin:
+                    if log_with_portnum:
+                        facade.logger.debug(
+                            f"Processed {portnum} with plugin {plugin.plugin_name}"
+                        )
+                    else:
+                        facade.logger.debug(f"Processed by plugin {plugin.plugin_name}")
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)